R/reactives.R

```diff
@@ -1583,6 +1708,20 @@ maskReactiveContext <- function(expr) {
 #' @param ignoreNULL Whether the action should be triggered (or value
 #'   calculated, in the case of \code{eventReactive}) when the input is
 #'   \code{NULL}. See Details.
+#' @param skipFirst Whether the action should be run the first time the
```

Comment:
It's not the first time it's invalidated. It's when it's first created. How about `ignoreInitial`? "When TRUE, the handler never runs when the observer is first created (even if the event expression returns a non-NULL value)."